l3kernel/doc/source3body.tex

```diff
@@ -588,6 +588,7 @@ \part{Data types}
     l3fp-functions.dtx
   }
 \ExplSyntaxOff
+\DocInput{l3bitset.dtx}
```

Comment:
Any better position than just between `fp` submodules and `fparray`?

Reply:
Moved